app/i18n/cz/admin.php

```diff
@@ -170,18 +170,18 @@
 		'registration' => array(
 			'number' => 'Maximální počet účtů',
 			'select' => array(
-				'label' => 'Registration form',	// TODO
+				'label' => 'Registrační formulář',
 				'option' => array(
-					'noform' => 'Disabled: No registration form',	// TODO
-					'nolimit' => 'Enabled: No limit of accounts',	// TODO
-					'setaccountsnumber' => 'Set max. number of accounts',	// TODO
+					'noform' => 'Zakazáno: Žádný registrační formulář',
+					'nolimit' => 'Povoleno: Bez omezení počtu účtů',
+					'setaccountsnumber' => 'Nastavit max. počet účtů',	// TODO also it can be: Nastavit maximální počet účtů
```

Comment:
A decision is needed here ;)

Reply:
I think `Nastavit maximální počet účtů` willl be better, if there is not an limit on length.

Reply:
It is enough space here :) 
![grafik](https://user-images.githubusercontent.com/1645099/193452629-ae8eeb02-1d28-4c83-a17d-a3fb1777efcf.png)

Reply:
Updated it to `Nastavit maximální počet účtů`